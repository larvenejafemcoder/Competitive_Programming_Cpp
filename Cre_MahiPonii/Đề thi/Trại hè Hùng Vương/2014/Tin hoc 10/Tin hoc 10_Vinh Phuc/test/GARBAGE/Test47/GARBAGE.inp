ENSWEEEDEWDWWSNDSWDDUEUUUEUNWNESDSSEDSD
EEWDNUNEEUEEENNDUDDWSWNSNWNDDNWENNWN
NNSUW
NDDUNUUNWEDDUUENDDDNNNWWUEUSUEUSDNNWNDNDWNESD
NDWNEUEENEEENWNSSNWDNSDNDNEDWSWESENSWUNEUNWNSSNDSDSENNNSUDUNEUUUDDEDSNEWW
UNESDWEUNNWUSSNEEDDUUDEUWEESWNEUNEWWENNNDWWW
D 60
