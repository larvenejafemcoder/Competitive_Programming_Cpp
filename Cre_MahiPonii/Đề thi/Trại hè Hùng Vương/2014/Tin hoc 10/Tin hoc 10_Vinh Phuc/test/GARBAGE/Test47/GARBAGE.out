62618437829004520117849274711715535147431768080181786940234785103765414268236656916909098837368
